src/micromamba/create.cpp

```diff
@@ -18,28 +18,16 @@ set_create_command(CLI::App* subcom)
     init_install_parser(subcom);
 
     subcom->callback([&]() {
-        auto& ctx = Context::instance();
-
         parse_file_options();
-        load_install_options(ctx);
+        load_configuration();
+
+        auto& configuration = Configuration::instance();
+        auto& specs = configuration.at("specs").value<std::vector<std::string>>();
 
-        if (!create_options.specs.empty() && !ctx.target_prefix.empty())
+        if (!specs.empty())
         {
-            if (ctx.target_prefix == ctx.root_prefix)
-            {
-                if (Console::prompt(
-                        "It's not possible to overwrite 'base' env. Install in 'base' instead?",
-                        'n'))
-                {
-                    install_specs(create_options.specs, false);
-                }
-                else
-                {
-                    exit(1);
-                }
-            }
-            catch_existing_target_prefix(ctx);
-            install_specs(create_options.specs, true);
+            check_target_prefix(false, false, true, false);
```

Comment:
hmmm ... this API doesn't speak to me. 

we could fake named arguments using `/*someparam=*/false, ...
I need to check the signature of check_target_prefix

Reply:
I'm not really happy with that either. I will try to improve it